docs/config-description.md

```diff
@@ -1,452 +1,592 @@
+
 - **Required**: False
 - **Type**: string
 - **Default value**: None
 - **Constraints**: None
- -   **Description**: Configuration file that contains the necessary security credentials and connection details for securely connecting to a Cassandra database cluster.
-### Key: database.cassandra.port
+- **Description**: The configuration file that contains the necessary credentials and connection details for securely connecting to a Cassandra database cluster.
+
+### database.cassandra.port
+
 - **Required**: False
 - **Type**: int
 - **Default value**: None
-- **Constraints**: The minimum value is `1`. The maximum value is `65535
- -   **Description**: Port number to connect to the database.
-### Key: database.cassandra.keyspace
+- **Constraints**: The minimum value is `1`. The maximum value is `65535`.
+- **Description**: The port number used to connect to the Cassandra database.
+
+### database.cassandra.keyspace
+
 - **Required**: True
 - **Type**: string
-- **Default value**: clio
+- **Default value**: `clio`
 - **Constraints**: None
- -   **Description**: Keyspace to use for the database.
-### Key: database.cassandra.replication_factor
+- **Description**: The Cassandra keyspace to use for the database. If you don't provide a value, this is set to `clio` by default.
+
+### database.cassandra.replication_factor
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 3
-- **Constraints**: The minimum value is `0`. The maximum value is `65535`
- -   **Description**: Number of replicated nodes for Scylladb. Visit this link for more details : https://university.scylladb.com/courses/scylla-essentials-overview/lessons/high-availability/topic/fault-tolerance-replication-factor/ 
-### Key: database.cassandra.table_prefix
+- **Default value**: `3`
+- **Constraints**: The minimum value is `0`. The maximum value is `65535`.
+- **Description**: Represents the number of replicated nodes for ScyllaDB. For more details see [Fault Tolerance Replication Factor](https://university.scylladb.com/courses/scylla-essentials-overview/lessons/high-availability/topic/fault-tolerance-replication-factor/).
+
+### database.cassandra.table_prefix
+
 - **Required**: False
 - **Type**: string
 - **Default value**: None
 - **Constraints**: None
- -   **Description**: Prefix for Database table names.
-### Key: database.cassandra.max_write_requests_outstanding
+- **Description**: An optional field to specify a prefix for the Cassandra database table names.
+
+### database.cassandra.max_write_requests_outstanding
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 10000
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: Maximum number of outstanding write requests. Write requests are api calls that write to database 
-### Key: database.cassandra.max_read_requests_outstanding
+- **Default value**: `10000`
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
```

Comment:
I think here and in many other places `0` as a minimum value doesn't make sense. Maybe we should switch the default minimum for number constaint to `1`? Do we have any place where `0` is a valid number?